src/metabase/metabot/settings.clj

```diff
@@ -38,7 +38,7 @@
 (defsetting is-metabot-enabled
   (deferred-tru "Is Metabot enabled?")
   :type :boolean
-  :visibility :authenticated
+  :visibility :public
```

Comment:
It is safe to expose and I need it for FE route guard to be present in the `html` template